### ISSUE TITLE
About the performance on ISIC2017 dataset

### ISSUE THREAD

**Issue by wangyxxjtu** [bigmb/Unet-Segmentation-Pytorch-Nest-of-Unets#34]
Hello, 
Great Work!
I have some questions about the results on ISIC2017(skin cancer segmentation dataset).  I trained the model using your code, and got satisfied results on DRIVE (eye ) and Lung segmentation datasets,  but I can't get normal results on ISIC 2017, the performance is very poor, the dc is only 0.34. So,  Do you have any idea about the problem?

**Comment by bigmb** [comment]
Thanks.
Did you check the intermediate training images and see if the segmentation is detecting the location of Gt correctly?

**Comment by wangyxxjtu** [comment]
Thank you for your reply. 
Following your suggestion, I randomly visualized several images and the corresponding masks during tranining, all these samples are exactly matched.

**Comment by bigmb** [comment]
Ok.
And all the Unets are showing the same results?
Did you try with Nested-Unet?

**Comment by wangyxxjtu** [comment]
Thank you.
I  found the problem, I made some changes on data transforms in training, learning to the inconsistent between the training data and the testing data. Unfortunately, after I fixed the data transform problem, the performance is still not reasonable, for unet on ISIC 2017, the DC is only 0.69 (0.86 reported in r2u-net paper), and the nestedu- net is only 0.73, which are much worse.

**Comment by bigmb** [comment]
Try to increase the depth of the model or decrease it.
Maybe that can provide a better dice score.

**Comment by wangyxxjtu** [comment]
Ok, thank you.
